### PR TITLE
fix(DB/quest_template): Update

### LINKED ISSUE #11393: Update quest_template
### Current Behaviour

Doing a comparison with TC I found several changes in the quest_template table

### Expected Blizzlike Behaviour

- ID: 183 LogDescription: (FIX https://github.com/azerothcore/azerothcore-wotlk/pull/11394/commits/810d9210677c5d791971943c7310ff3b9637bfde)

ACore: Talin Keeneye would like you to kill 8 Small Crag Boars.
TCore: Talin Keeneye would like you to kill 12 Small Crag Boars.

- ID: 287 QuestCompletionLog: _(OK no change in AC)_

ACore: Return to Senir Whitebeard at Kharanos in Dun Morogh.
TCore: Return to Senir Whitebeard at Shimmer Ridge in Dun Morogh.

- ID: 317 QuestCompletionLog: _(OK no change in AC)_

ACore: Return to Pilot Bellowfiz at Steelgrill's in Dun Morogh.
TCore: Return to Pilot Bellowfiz at The Tundrid Hills in Dun Morogh.

- ID: 416 QuestCompletionLog: _(OK no change in AC)_

ACore: Return to Mountaineer Kadrell at Thelsamar in Loch Modan.
TCore: Return to Mountaineer Kadrell at Grizzlepaw Ridge in Loch Modan.

- ID: 543 QuestCompletionLog: _(OK no change in AC)_

ACore: Return to Count Remington Ridgewell at Stormwind Keep in Stormwind City
TCore: Return to Count Remington Ridgewell at Northshire Valley in Elwynn Forest.

- ID: 806 QuestCompletionLog: (FIX https://github.com/azerothcore/azerothcore-wotlk/pull/11394/commits/4ca6f1bf800c2b8c1e2750041af0fc85094bf8b2)

ACore: Did you find Fizzle, $N? He, and rest of the Burning Blade, must be scoured from our lands!
TCore: Return to Orgnil Soulscar in Durotar.

- ID: 932 QuestCompletionLog: _(OK no change in AC)_

ACore: Return to Tallonkai Swiftroot at Dolanaar in Teldrassil.
TCore: Return to Tallonkai Swiftroot at Lake Al'Ameth in Teldrassil.

- ID: 1013 QuestCompletionLog: _(OK no change in AC)_

ACore: Return to Keeper Bel'dugur at Apothecarium in the Undercity.
TCore: Return to Keeper Bel'dugur at Lordamere Lake in Alterac Mountains.

- ID: 1195 QuestCompletionLog: _(OK no change in AC)_

ACore: Return to Zangen Stonehoof in Thunder Bluff.
TCore: Return to Zangen Stonehoof at The Golden Plains in Mulgore.

- ID: 1886 QuestCompletionLog: _(OK no change in AC)_

ACore: Return to Mennet Carkad at Undercity in the Rogues' Quarter.
TCore: Return to Mennet Carkad at Ruins of Lordaeron in Tirisfal Glades.

- ID: 1899 QuestCompletionLog: _(OK no change in AC)_

ACore: Return to Mennet Carkad at Undercity in the Rogues' Quarter.
TCore: Return to Mennet Carkad at Ruins of Lordaeron in Tirisfal Glades.

- ID: 2199 QuestCompletionLog: _(OK no change in AC)_

ACore: Return to Talvash del Kissel at the Mystic Ward in Ironforge.
TCore: Return to Talvash del Kissel in Wetlands.

- ID: 2204 QuestCompletionLog: _(OK no change in AC)_

ACore: Return to Talvash del Kissel at the Mystic Ward in Ironforge.
TCore: Return to Talvash del Kissel in Wetlands.

- ID: 2459 QuestCompletionLog: _(OK no change in AC)_

ACore: Return to Tallonkai Swiftroot at Dolanaar in Teldrassil.
TCore: Return to Tallonkai Swiftroot at Lake Al'Ameth in Teldrassil.

- ID: 5283 QuestCompletionLog: _(OK no change in AC)_

ACore: Return to Grumnus Steelshaper in Ironforge.
TCore: Return to Grumnus Steelshaper in Dun Morogh.

- ID: 6024 QuestCompletionLog: _(OK no change in AC)_

ACore: Return to Mound of Dirt in The Eastern Plaguelands
TCore: Return to Mound of Dirt in The Hinterlands.

- ID: 6561 QuestCompletionLog :_(OK no change in AC)_

ACore: Return to Bashana Runetotem at Elder Rise in Thunder Bluff.
TCore: Return to Bashana Runetotem at The Golden Plains in Mulgore.

- ID: 7461 QuestCompletionLog: _(OK no change in AC)_

ACore: Return to Shen'dralar Ancient in Dire Maul.
TCore: Return to Shen'dralar Ancient at Dalaran Crater in Alterac Mountains.

- ID: 8355 QuestCompletionLog: _(OK no change in AC)_

ACore: Return to Talvash del Kissel at the Mystic Ward in Ironforge.
TCore: Return to Talvash del Kissel in Wetlands.

- ID: 9339 LogDescription: _(OK no change in AC)_

ACore: Return the Flame of Stormwind to (NAME).
TCore:

- ID: 9339 QuestDescription: _(OK no change in AC)_

ACore: This flame, stolen from the heart of Stormwind itself, floods you with warmth. This raw power is foreign to you, but the (NAME) might know more...
TCore:

- ID: 9365 LogDescription: _(OK no change in AC)_

ACore: Return the Flame of Stormwind to (NAME).
TCore:

- ID: 9365 QuestDescription: _(OK no change in AC)_

ACore: This flame, stolen from the heart of Stormwind itself, floods you with warmth. This raw power is foreign to you, but the (NAME) might know more...
TCore:

- ID: 12032 AreaDescription: _(OK no change in AC)_

ACore:
TCore: Oacha'noa's compulsion obeyed.

- ID: 12032 ObjectiveText1: _(OK no change in AC)_

ACore: Oacha'noa's compulsion obeyed
TCore:

- ID: 14351 QuestCompletionLog: _(OK no change in AC)_

ACore: Return to Bragor Bloodfist at the Royal Quarter in the Undercity.
TCore: Return to Bragor Bloodfist at Lordamere Lake in Silverpine Forest.

- ID: 14352 QuestCompletionLog: _(OK no change in AC)_

ACore: Return to Bragor Bloodfist at the Royal Quarter in the Undercity.
TCore: Return to Bragor Bloodfist at Lordamere Lake in Silverpine Forest.

- ID: 14356 QuestCompletionLog: _(OK no change in AC)_

ACore: Return to Bragor Bloodfist at the Royal Quarter in the Undercity.
TCore: Return to Bragor Bloodfist at the Royal Quarter in Undercity.

- ID: 14418 QuestCompletionLog: _(OK no change in AC)_

ACore: Return to Aleric Hawkins at Undercity in the Hall of the Dark Lady.
TCore: Return to Aleric Hawkins at Lordamere Lake in Silverpine Forest.

- ID: 14419 QuestCompletionLog: _(OK no change in AC)_

ACore: Return to Andron Gant at Undercity in the Apothecarium.
TCore: Return to Andron Gant at Lordamere Lake in Alterac Mountains.

- ID: 14420 QuestCompletionLog: _(OK no change in AC)_

ACore: Return to Mennet Carkad at Undercity in the Rogues' Quarter.
TCore: Return to Mennet Carkad at Ruins of Lordaeron in Tirisfal Glades.

- ID: 14421 QuestCompletionLog: _(OK no change in AC)_

ACore: Return to Mennet Carkad at Undercity in the Rogues' Quarter.
TCore: Return to Mennet Carkad at Ruins of Lordaeron in Tirisfal Glades.

### Source

TrinityCore Database 3.3.5

### Steps to reproduce the problem

*

### Extra Notes

I will check the differences one by one, with external links like wowhead, and other pages

### AC rev. hash/commit

AzerothCore rev. 4ac565699fcb 2022-04-13 11:57:04 +0000 (master branch) (Unix, RelWithDebInfo, Static)

### Operating system

Ubuntu 20.04

### Custom changes or Modules

_No response_

### PR DESCRIPTION
**Pull request by Tony931023**
### Update Quest: #11393 
- ID 183 - LogDescription
- ID 806 - QuestCompletionLog
- Close #11393 

### Update RewardNextQuest

1. https://www.wowhead.com/quest=6611/to-gadgetzan-you-go (Horde)  NextQuest: 6610 
2. https://www.wowhead.com/quest=6612/i-know-a-guy (Alliance) NextQuest: 6610
3. https://www.wowhead.com/quest=8335/felendren-the-banished  (Horde)  NextQuest: 8347
4. https://www.wowhead.com/quest=9280/replenishing-the-healing-crystals (Alliance) NextQuest: 9409
5. https://wotlkdb.com/?quest=9293 (Alliance) NextQuest: 9294
6. https://www.wowhead.com/quest=9705/package-recovery  (Horde)  NextQuest: 8350
7. https://wotlkdb.com/?quest=11160  (Horde - Alliance )  NextQuest: none
8. https://www.wowhead.com/quest=11571/learning-to-communicate (Horde - Alliance )  NextQuest: 11559
9. https://www.wowhead.com/quest=11662/seek-out-karuk (Horde)  NextQuest: none
10. https://wotlkdb.com/?quest=12249 (Alliance) NextQuest: 12070
11. https://wotlkdb.com/?quest=12598 (Horde - Alliance )  NextQuest: 12555
12. https://wotlkdb.com/?quest=12606 (Horde - Alliance )  NextQuest: none
13. https://wotlkdb.com/?quest=12880 (Alliance) NextQuest: none
14. https://www.wowhead.com/quest=12974/the-champions-call (Horde - Alliance )  NextQuest: 12932
15. https://wotlkdb.com/?quest=13005 (Horde - Alliance )  NextQuest: none
16. https://www.wowhead.com/quest=13593/valiant-of-stormwind (Alliance) NextQuest: 13718

### RewardMoney 
Quest ID 3644, 3645, 3646, 3647 Name: Membership Card Renewal
https://wotlkdb.com/?quest=3644
https://wotlkdb.com/?quest=3645
https://wotlkdb.com/?quest=3646
https://wotlkdb.com/?quest=3647

 *  Required money: 2 ore (-20000)

### RewardItem1 and RewardAmount1 
https://www.wowhead.com/quest=9561/nolkais-words  ITEM:  [Nolkai's Box](https://www.wowhead.com/item=23846/nolkais-box) ID: 23846 Amount: 1
https://www.wowhead.com/quest=10172/there-is-no-hope#  ITEM: [Insignia of the Mag'hari Hero](https://www.wowhead.com/item=28168/insignia-of-the-maghari-hero) ID: 28168 Amount: 1

### CODE REVIEW

**Comment by Nyeriah** [comment]
@Tony931023 Please try to group the translations in a single PR, so we don't have several PRs with one line each. Just keep them at a reasonable size. Thank you very much!

**Comment by Tony931023** [comment]
If that's why the PR is in Draft, I will review each one and update the PR

File: data/sql/updates/pending_db_world/rev_1649856189895360036.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1649856189895360036');
+
+SET @ID := 183;
+UPDATE `quest_template` SET `LogDescription` = 'Talin Keeneye would like you to kill 12 Small Crag Boars.'  WHERE `ID` = @ID;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1649856189895360036.sql:4]: Please don't make variables for one liners as you won't reuse this ID

**Review by Kitzunu — DISMISSED** [review]

**Comment by Tony931023** [comment]
Ready

**Comment by Tony931023** [comment]
ready to be reviewed

**Comment by Lynethris** [comment]
Tested and it's fine.

File: data/sql/updates/pending_db_world/rev_1649856189895360036.sql [hunk 2]
```diff
@@ -0,0 +1,25 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1649856189895360036');
+
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1649856189895360036.sql:2]: can u remove the version_db_world line please? there was a change a few days ago and we no longer require this line.

**Review by acidmanifesto — CHANGES_REQUESTED** [review]

**Review by acidmanifesto — APPROVED** [review]

**Comment by acidmanifesto** [comment]
@Kitzunu is the changes to your liking?

**Comment by acidmanifesto** [comment]
Thank you for the PR.

**Pull request merged**